change/@uifabric-charting-2020-03-30-15-55-09-user-v-sivsar-HideLegendInCharts.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
+  "comment": "adding new prop hideLegend in donut chart and line chart which decides to show or hide legends",
```

Comment:
Revise comment so it follows our release note structure. "DonutChart: add new prop hideLegend."